core/trino-main/src/test/java/io/trino/operator/window/WindowAssertions.java

```diff
@@ -22,77 +22,78 @@
+                ( 6, 'F', '1992-02-21'),
+                ( 7, 'O', '1996-01-10'),
+                (32, 'O', '1995-07-16'),
+                (33, 'F', '1993-10-27'),
+                (34, 'O', '1998-07-21')
+            ) AS orders (orderkey, orderstatus, orderdate)""";
+
+    private static final String VALUES_WITH_NULLS = """
+            SELECT *
+            FROM (
+              VALUES
+                ( 1,                   CAST(NULL AS VARCHAR), CAST(NULL AS VARCHAR)),
+                ( 3,                   'F',                   '1993-10-14'),
+                ( 5,                   'F',                   CAST(NULL AS VARCHAR)),
+                ( 7,                   CAST(NULL AS VARCHAR), '1996-01-10'),
+                (34,                   'O',                   '1998-07-21'),
+                ( 6,                   'F',                   '1992-02-21'),
+                (CAST(NULL AS BIGINT), 'F',                   '1993-10-27'),
+                (CAST(NULL AS BIGINT), 'O',                   '1996-12-01'),
+                (CAST(NULL AS BIGINT), CAST(NULL AS VARCHAR), CAST(NULL AS VARCHAR)),
+                (CAST(NULL AS BIGINT), CAST(NULL AS VARCHAR), '1995-07-16')
+            ) AS orders (orderkey, orderstatus, orderdate)""";
+
+    private static final String VALUES_WITH_NAN = """
+            SELECT *
+            FROM (
+              VALUES
+                ( 1, 'O', '1996-01-02'),
+                ( 2, 'O', '1996-12-01'),
+                ( 3, 'F', '1993-10-14'),
+                ( 4, 'O', '1995-10-11'),
+                ( nan(), 'F', '1994-07-30'),
+                ( 6, 'F', '1992-02-21'),
+                ( 7, 'O', '1996-01-10'),
+                (32, 'O', '1995-07-16'),
+                (33, 'F', '1993-10-27'),
+                (34, 'O', '1998-07-21')
+            ) AS orders (orderkey, orderstatus, orderdate)""";
+
+    private static final String VALUES_WITH_INFINITY = """
+            SELECT *
+            FROM (
+              VALUES
+                ( 1, 'O', '1996-01-02'),
+                ( 2, 'O', '1996-12-01'),
+                ( 3, 'F', '1993-10-14'),
+                ( 4, 'O', '1995-10-11'),
+                ( infinity(), 'F', '1994-07-30'),
+                ( 6, 'F', '1992-02-21'),
+                ( 7, 'O', '1996-01-10'),
+                (32, 'O', '1995-07-16'),
+                (33, 'F', '1993-10-27'),
+                (34, 'O', '1998-07-21')
+            ) AS orders (orderkey, orderstatus, orderdate)""";
 
     private WindowAssertions() {}
 
     public static void assertWindowQuery(@Language("SQL") String sql, MaterializedResult expected, QueryRunner queryRunner)
     {
-        @Language("SQL") String query = format("" +
-                "SELECT orderkey, orderstatus,\n%s\n" +
-                "FROM (%s) x", sql, VALUES);
+        @Language("SQL") String query = format("""
+                SELECT orderkey, orderstatus,
+                %s
+                FROM (%s) x""", sql, VALUES);
```

Comment:
For multiline statements, place arguments in separate lines:

```suggestion
        @Language("SQL") String query = format(
            """
            SELECT orderkey, orderstatus, %s
            FROM (%s) x
            """, 
            sql, 
            VALUES);
```